cypress/e2e/default/settings/username-change.ts

```diff
@@ -5,7 +5,8 @@ describe('Username input field', () => {
   });
 
   it('Should be possible to type', () => {
-    cy.typeUsername('twaha').should('have.attr', 'value', 'twaha');
+    cy.typeUsername('twaha');
+    cy.get('@usernameInput').should('have.attr', 'value', 'twaha');
```

Comment:
This is mostly out of my own interest:

Why do we have a mixture of this and?:

```ts
cy.get('@usernameForm')
```

Reply:
So this was the compiler error:

![Screen Shot 2023-01-26 at 9 57 49 AM (2)](https://user-images.githubusercontent.com/108169988/214916667-5025ae41-ca44-43d0-87a9-0a742bf7bd83.png)

When I went to demo the problem, I realized I'd missed a better solution. I thought the compiler was angry because I was trying to chain a `.should` after a `.type`—which makes no sense—but I just needed to annotate the command. 🤦‍♂️

The new solution is passing on my machine and I pushed the change up.